form-los-altos-urgent-report.html

```diff
@@ -0,0 +1,135 @@
+          <td rowspan="4">
+            <textarea name="23.description" rows="5" required="required"></textarea>
+          </td>
+          <td class="label">Time:</td>
+          <td>
+            <input name="1b.time" class="" type="text" size="10"
+                   required="required" pattern="\d{1,2}:\d{2}(:\d{2})?"
+                   value="{{time}}"/>
+          </td>
+        </tr><tr>
+          <td class="label">BAT #</td>
+          <td><input name="20.BAT" type="text" required/></td>
+        </tr><tr>
+          <td class="label">Reported by:</td>
+          <td><input name="21.reporter" type="text"/></td>
+        </tr><tr>
+          <td class="label">Their Msg #</td>
+          <td>
+            <input name="2.txmsgno" type="text" readOnly="true"
+                   value="{{query-string:txmsgno}}"/>
+          </td>
+        </tr>
+      </table>
+      <table>
+        <tr>
+          <td class="label" style="text-align:left;vertical-align:top;">Notes/Actions:
+            <textarea name="24.notes" rows="6"></textarea>
+          </td>
+        </tr>
+      </table>
+      <br/>
+      <table class="noborder">
+        <tr>
+          <td class="label">Severity:</td>
+          <td>
+            <select name="4.severity">
+              <option value="EMERGENCY">Emergency (life threat)</option>
+              <option value="URGENT">Urgent (property threat)</option>
+              <option value="OTHER">Other</option>
+            </select>
+          </td>
+        </tr><tr>
+          <td class="label">Handling:</td>
+          <td>
+            <select name="5.handling">
+              <option value="IMMEDIATE">Immediate (as soon as possible)</option>
+              <option value="PRIORITY">Priority (less than one hour)</option>
+              <option value="ROUTINE">Routine (more than one hour)</option>
+            </select>
+          </td>
+        </tr><tr>
+          <td class="label" style="vertical-align:bottom;">Deliver to:</td>
+          <td>
+            <fieldset class="horizontal">
+              <label>
+                <input name="7a.toMgt" type="checkbox" value="Yes"/>Management
+              </label><label>
+                <input name="7b.toOps" type="checkbox" value="Yes"/>Operations
+              </label><label>
+                <input name="7c.toPlan" type="checkbox" value="Yes"/>Planning
+              </label><label>
+                <input name="7d.toLog" type="checkbox" value="Yes"/>Logistics
+              </label><label>
+                <input name="7e.toFin" type="checkbox" value="Yes"/>Finance
+              </label>
+            </fieldset>
+          </td>
+        </tr>
+      </table>
+    </form>
+    <div data-include-html="outpost_message_header"></div>
+    <div data-include-html="submit-buttons">{
+      "4.": "URGENT",
+      "5.": "PRIORITY",
+      "7b.toOps": "checked",
+      "7c.toPlan": "checked"}
+    </div>
```

Comment:
Does this JSON object have any effect? I wouldn't expect it to since the submit-buttons include fragment does not include any of these fields.

Reply:
It causes initialization of some fields which are defined here in the same file. Perhaps it's puzzling that the fields aren't defined in submit-buttons.

The same effect could be achieved with a short <script> that calls init_form_from_fields.